composer.json

```diff
@@ -32,7 +32,7 @@
         "doctrine/inflector": "^1.4 || ^2.0",
         "doctrine/instantiator": "^1.3",
         "doctrine/lexer": "^1.2.3",
-        "doctrine/persistence": "^2.4",
+        "doctrine/persistence": "^2.5@dev",
```

Comment:
TODO: Revert this live as soon as Persistence 2.5.0 has been released.

Reply:
Let's release persistence 2.5? Instead of risking forgetting about this / blocking 2.12.0?

Reply:
I've released 2.5.